src/postprocess/kinetic_energy_density.jl

```diff
@@ -0,0 +1,20 @@
+"""
+Computes the kinetic energy density (ked), 1/2 ∑ fn |∇ψn|^2.
+"""
+function compute_kinetic_energy_density(basis::PlaneWaveBasis, ψ, occupation)
+    @assert all(symop -> length(symop) == 1, basis.ksymops) == 1  # TODO lift this
+    ked = zeros(eltype(basis), basis.fft_size)
+    for (ik, kpt) in enumerate(basis.kpoints)
+        for (n, ψnk) in enumerate(eachcol(ψ[ik]))
+            for α = 1:3
+                dαψnk = [im*(G+kpt.coordinate_cart)[α] for G in G_vectors_cart(kpt)] .* ψnk

```

Comment:
We try to keep one space around operators.

```suggestion
                dαψnk = [im * (G + kpt.coordinate_cart)[α] for G in G_vectors_cart(kpt)] .* ψnk
```